pandas/io/gbq.py

```diff
@@ -682,6 +692,15 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.19.0
 
+    **kwargs: Arbitrary keyword arguments
+        configuration (dict): query config parameters for job processing.
+            For example:
+                configuration = {'query': {'useQueryCache': False}}
```

Comment:
there need to be a blank line before and after this line (again, this are some sphinx (rst) syntax things)